docs/releases/2.0.txt

```diff
@@ -326,6 +326,12 @@ Miscellaneous
   require them. Third-party apps that use these methods may want to adopt a
   similar approach.
 
+* The :func:`django.views.decorators.http.condition` decorator as well as
+  :func:`django.views.decorators.http.etag` and
+  :func:`django.views.decorators.http.last_modified` which build off of it
+  no longer return ``ETag`` and ``Last-Modified`` headers for non-safe
+  methods. Set them in your view if necessary.
```

Comment:
"return" is possibly ambiguous when we're talking about a Python function. How about: "...no longer add ``ETag`` and ``Last-Modified`` headers to the response for non-safe request methods."